test/Tests/src/Runtime/Stack_Traces_Spec.enso

```diff
@@ -4,17 +4,17 @@ import Standard.Test
 type My_Type
 
 bar = Runtime.get_stack_trace
-baz = here.bar
-Number.foo = here.baz
+baz = bar
+Number.foo = baz
 foo x = x.foo
-My_Type.foo = here.foo 123
+My_Type.foo = foo 123
 
 spec = Test.group "Stack traces" <|
     Test.specify "should capture traces correctly" <|
-        modname = Meta.Constructor (Meta.meta here . constructor) . name
+        modname = Meta.Constructor (Meta.meta self . constructor) . name
```

Comment:
This change seems to change the semantics, I think the original code was purposefully doing `(Meta.meta self).constructor` because the `constructor` is a method on the `Meta` type.

Reply:
Does this code work correctly? Looks slightly suspicious. Cannot we use `Meta.meta Stack_Traces_Spec . constructor`? With `here` removed, I guess we still need some way to refer to the current module, right?

Reply:
Changes to spec name. Thanks for the suggestion, @radeusgd!